examples/code_repository/README.md

```diff
@@ -0,0 +1,93 @@
+# Using a code repository
+
+Code repositories enable ZenML to keep track of the code version that you use for your
+pipeline runs. Additionally, running a pipeline which is tracked in a registered code
+repository can speed up the Docker image building for containerized stack components.
+
+## 📄 Prerequisites
+
+In order to run your ZenML pipelines with a code repository, we need to set up a 
+few things first:
+
+* First you'll need a [GitHub](https://github.com) account and a GitHub repository
+which contains code to run a ZenML pipeline. If you're new to ZenML and don't have such
+a repository yet, we suggest you start by forking
+[this example repository](https://github.com/zenml-io/code-repository-example) to get
+started quickly. To do so, visit the repository page and click on `Fork` in the top right,
+after which you'll be able to create a new fork in your personal GitHub account. Once that
+is complete, clone the repository to your local machine.
+* You'll also need to create a GitHub personal access token that has read permissions for
+the repositories in your account. To do so, please follow [this guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) 
+and make sure to assign your token the **repo** scope.
+
+If you also want to test the Docker build speedup that is provided by using a
```

Comment:
maybe move this into the lower section so its not front loaded with pre-requisites